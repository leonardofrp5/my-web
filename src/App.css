.container {
  align-items: center;
  background-color: #282C34;
  display: flex;
  height: 100vh;
  justify-content: center;
  width: 100%;
}
.title {
  color: #61DAFB;
  font-size: 7em;
}

@media screen and (max-width: 1024px) {
  .title {
    font-size: 4em;
  }
}

@media screen and (max-width: 600px) {
  .title {
    font-size: 2em;
  }
}
