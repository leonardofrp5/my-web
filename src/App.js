import React from 'react';

import Home from 'page/application/Home';

function App() {
  return (
    <>
      <Home />
    </>
  );
}

export default App;
