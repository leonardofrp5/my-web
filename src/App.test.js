import { render } from '@testing-library/react'

import App from './App'

describe('#App', () => {
  test('Check component App', () => {
    const { container } = render(<App />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
