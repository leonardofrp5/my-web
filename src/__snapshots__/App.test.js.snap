// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#App Check component App 1`] = `
<div
  class="container"
>
  <img
    alt="img_leonardo"
    class="imgProfile"
    src="imgProfile.png"
  />
  <h3
    class="title"
  >
    Leonardo Romero Perez
  </h3>
</div>
`;
